packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -70,6 +71,14 @@ async function run(options: {
   if (!fs.existsSync(srcDir)) {
     throw new Error(`--source path does not exist: ${srcDir}.`);
   }
+  const persistModule = options.persist && path.resolve(process.cwd(), options.persist);
+  if (persistModule && !fs.existsSync(persistModule)) {
+    throw new Error(`--persist path does not exist: ${persistModule}.`);
+  }
+  // Using eval here to prevent webpack from trying to rewrite the require and
+  // failing.
+  const persistQuery = persistModule &&
+    eval(`require(${JSON.stringify(persistModule)})`);
```

Comment:
Seems like it does work (at least, build runs without blowing up); webpack lets the line in question through as:

```
var persistQuery = persistModule && global['require'](persistModule);
```

in the built output.